export.sh

```diff
@@ -57,6 +116,8 @@ if [ "$RUN_EXPORT" = "y" ] || [ "$RUN_EXPORT" = "Y" ]; then
     exit 1
   fi
 
+  echo "$EXPORT_FILE"
+
   echo "Done. Export has been saved to $(pwd)/$EXPORT_FILE"
```

Comment:
The additional `echo "$EXPORT_FILE"` seems like a duplication of the line below.